scripts/replace-all-mocks.py

```
@@ -0,0 +1,169 @@
+#!/usr/bin/env python3
+"""Replace all mock implementations with real endpoints"""
+
+import re
+from pathlib import Path
+from typing import List, Tuple
```

Ruff (0.11.9)

`typing.List` is deprecated, use `list` instead

(UP035)